quantum/process_keycode/process_leader.c

```diff
@@ -49,24 +49,24 @@ bool process_leader(uint16_t keycode, keyrecord_t *record) {
   // Leader key set-up
   if (record->event.pressed) {
     if (leading) {
-      if (timer_elapsed(leader_time) < LEADER_TIMEOUT) {
-#ifndef LEADER_KEY_STRICT_KEY_PROCESSING
+    //   if (timer_elapsed(leader_time) < LEADER_TIMEOUT) {
+        #ifndef LEADER_KEY_STRICT_KEY_PROCESSING
```

Comment:
Please revert this formatting change:
```suggestion
#ifndef LEADER_KEY_STRICT_KEY_PROCESSING
```

Reply:
I think that it improves readability, but ok, I'll make it look like the rest of the codebase